lib/internal/bridge/platform_to_dart_bridge.dart

```diff
@@ -14,7 +13,10 @@ class PlatformToDartBridge {
   Future<dynamic> handleCall(MethodCall call) async {
     print("Observed method call on Flutter Simulator: ${call.method}");
     if (_isCallCancellable(call)) {
-      return _handleCancelablePlatformCall(call);
+      return _manager.handleCancelablePlatformCall(
```

Comment:
Bridge shouldn't be aware of this. Bridge should just parse method name, unwrap arguments and pass those to the manager and it should handle any additional logic concerning ability to be canceled.